### ISSUE TITLE
Vite module federation

### ISSUE THREAD

**Issue by drodmun** [goveo/react-international-phone#73]
### Discussed in https://github.com/goveo/react-international-phone/discussions/70

<div type='discussions-op-text'>

<sup>Originally posted by **raffidahmad** August  9, 2023</sup>
Cant seem to get it to work with module federation getting this error and unable to identify the cause
```
Uncaught TypeError: r is null
    Qr React
    usePhoneInput __federation_expose_UserManagement-f2d6de70.js:1040
    PhoneInput __federation_expose_UserManagement-f2d6de70.js:1040
    React 15
    notifyComponents index.js:4246
    replaceState index.js:4544
    applyActionsToStore index.js:2874
    queueOrPerformStateUpdate index.js:2898
    setRecoilValue index.js:2941
    useSetRecoilState index.js:5388
    g __federation_expose_UserManagement-f2d6de70.js:1028
    b __federation_expose_UserManagement-f2d6de70.js:1028
    onClick 
```</div>

**Comment by goveo** [comment]
Duplicate of #74

**Comment by goveo** [comment]
@drodmun This issue was fixed in v3.0.1
Consider leaving a star on this repo if you like the project. ⭐ Thanks!
